src/components/BlurImg.js

```diff
@@ -0,0 +1,32 @@
+import React, { useState } from "react";
+import { useEffect } from "react";
```

Comment:
You can just import both useState and useEffect in single import 
```
import React,  {useState, useEffect} from react
``` 
or if you want to add two imports for default and named import  you can do 
```
import React from 'react;
import {useState, useEffect} from 'react';
```

Reply:
Yaah! I know but that is happened because I installed react snippets extension in my dev environment and sometimes auto imports make the code 1 line longer